docs/sources/logql/_index.md

```diff
@@ -4,29 +4,27 @@ weight: 700
 ---
 # LogQL: Log Query Language
 
-Loki comes with its own PromQL-inspired language for queries called *LogQL*.
-LogQL can be considered a distributed `grep` that aggregates log sources.
+LogQL is Loki's PromQL-inspired query language.
+Queries act as if they are a distributed `grep` to aggregate log sources.
```

Comment:
I'm sorry for the nit picking. Should we format Markdown files with newlines or let the editor take care of line breaks? A former colleague convinced me at some point that it makes sense to write this without the line break :shrug:

Reply:
I believe I'm the editor, and I explicitly put in line breaks at the end of phrases and sentences. I'm not sure what your colleague's argument was, but I put them in because it is ever so much easier to edit at the sentence level. Markdown doesn't get in the way of having the line breaks, and neither does github's suggestion mechanism.

How about if we talk offline about the pros and cons of having line breaks?